chrome/skin/style-osx.css

```diff
@@ -0,0 +1,5 @@
+#universal-search-recommendation {
+  background-color: Highlight;
+  background-image: linear-gradient(rgba(255,255,255,0.3), transparent);
+  color: HighlightText;
+}/n
```

Comment:
Let's get rid of /n and adjust your editor so that newlines are auto-inserted :+1: